### DISCUSSION TITLE
Allow Ombi to communicate with both Plex and Sonarr behind Gluetun

### DISCUSSION THREAD

**Discussion by MrGoodGuy2144** [qdm12/gluetun#1465] [Q&A] [answered]
I am currently running a stack with sonarr/prowlarr/radarr/qbit behind gluetun by using network_mode: "service:gluetun". My plex/ombi containers and not in the same stack therefore not in the same network, but ombi needs to connect to both plex and sonarr/radarr. If i put ombi in the same stack as the other arr's, then it is unable to connect to the plex container. If i leave ombi outside the arr's stack then i cannot connect to sonarr/radarr. What would be the most easy solution to this?

Ive considered removing radarr/sonarr/prow from the stack and using the proxy option inside each container to connect to gluetun, but then radarr/sonarr/prow cannot connect to qbittorrent that is behind the vpn container.

From my understanding all it would take is a firewall rule change on the gluetun container to allow Ombi to talk with plex if the ombi container is using the vpn container. Not sure the best way to do this.

**Comment by bnhf** [comment, marked as answer]
@MrGoodGuy2144 

> If i leave ombi outside the arr's stack then i cannot connect to sonarr/radarr.

I'm not doing the Ombi-Plex watchlist integration part, however, I'm not seeing the same behavior as far as Ombi communicating with the *arrs. Our setups seem similar, as I have Gluetun and the *arrs in one stack with Plex and Ombi on another machine. In my case Plex is installed natively on Windows and Ombi is running under Docker Desktop on that same machine. Ombi requests are honored, and the results show up in Plex. Hostname - Port - API key for the *arrs are all that's required in Ombi -- if you can access their WebUIs on those same ports from another machine it seems like you should be good. Is there more to the story?

**Reply by MrGoodGuy2144** [reply]
Thanks for the response! While i didn't quite fix the problem i mentioned, i didn't actually realize that connecting ombi to plex was only needed for the watchlist integration part (i'm new to ombi). I dont mind putting ombi in with the Arr's stack and not being able to communicate with plex as long as it can see the all of my content via the sonarr/radarr library. The only thing that does slightly suck is i have a few libraries in plex that are not located in sonarr because they are hosted on a cloud provider mounted with rclone, this means unless i index my entire cloud library in sonarr it will not be able to tell if i have that content available already or not... Not a huge deal, but a little annoying because i will have to make the user confirm the movie/tv is not already on plex before searching ombi for it (because 6x the amount of movies of my clouddrive will not be indexed by ombi). 

Regardless, this will work good enough for my needs i appreciate the help.

One thing i will mention, the difference between our setup's is i have all of my containers on one Ubuntu 22.04 machine. I can access all the webui's from another PC just fine, and i suspect if i was running ombi on a separate machine it would be able to connect to the arr's just fine because i am able to access the arr's from another pc. I think the issue is related to them being run on the same machine.
